### ISSUE TITLE
Exception after upgrade: ExpectedAuthenticatedLndGrpcForGetInfoRequest

### ISSUE THREAD

**Issue by Kixunil** [apotdevin/thunderhub#124]
**Describe the problem/bug**
After upgrading from 0.8.13 to 0.9.3 ThunderHuh glitches (momentarily shows failed SSO, then it appears, then there's a long loading...) and then starts working properly.

<details>
<summary>Log</summary>

```
Aug 13 17:40:29 polisnode systemd[1]: Started Lightning Node Manager.
Aug 13 17:40:31 polisnode thunderhub[22654]: ready - started server on http://localhost:4000
Aug 13 17:51:11 polisnode thunderhub[22654]: 2020-08-13 17:51:11 error [THUB]: [ 400,
Aug 13 17:51:11 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndGrpcForGetInfoRequest',
Aug 13 17:51:11 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:51:11 polisnode thunderhub[22654]: 2020-08-13 17:51:11 error [THUB]: [ 400,
Aug 13 17:51:11 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndToRetrieveChainBalance',
Aug 13 17:51:11 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:51:11 polisnode thunderhub[22654]: 2020-08-13 17:51:11 error [THUB]: [ 400, 'ExpectedLndForPendingChainBalance', [length]: 2 ]
Aug 13 17:51:11 polisnode thunderhub[22654]: 2020-08-13 17:51:11 error [THUB]: [ 400, 'ExpectedLndGrpcApiForChannelBalanceQuery', [length]: 2 ]
Aug 13 17:51:11 polisnode thunderhub[22654]: { Error: ExpectedAuthenticatedLndGrpcForGetInfoRequest
Aug 13 17:51:11 polisnode thunderhub[22654]: ExpectedAuthenticatedLndToRetrieveChainBalance
Aug 13 17:51:11 polisnode thunderhub[22654]: ExpectedLndForPendingChainBalance
Aug 13 17:51:11 polisnode thunderhub[22654]: ExpectedLndGrpcApiForChannelBalanceQuery
Aug 13 17:51:11 polisnode thunderhub[22654]:     at new ApolloError (/usr/lib/thunderhub/node_modules/@apollo/client/errors/errors.cjs.js:31:28)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at /usr/lib/thunderhub/node_modules/@apollo/client/core/core.cjs.js:1385:47
Aug 13 17:51:11 polisnode thunderhub[22654]:     at /usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:921:69
Aug 13 17:51:11 polisnode thunderhub[22654]:     at new Promise (<anonymous>)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at Object.next (/usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:921:21)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at notifySubscription (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:135:18)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at onNotify (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:179:3)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at SubscriptionObserver.next (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:235:7)
Aug 13 17:51:11 polisnode thunderhub[22654]:     at /usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:909:68
Aug 13 17:51:11 polisnode thunderhub[22654]:     at Array.forEach (<anonymous>)
Aug 13 17:51:11 polisnode thunderhub[22654]:   graphQLErrors:
Aug 13 17:51:11 polisnode thunderhub[22654]:    [ { Error: ExpectedAuthenticatedLndGrpcForGetInfoRequest
Aug 13 17:51:11 polisnode thunderhub[22654]:          at promise.then.catch.err (/usr/lib/thunderhub/.next/server/pages/api/v1.js:995:11)
Aug 13 17:51:11 polisnode thunderhub[22654]:          at process._tickCallback (internal/process/next_tick.js:68:7)
Aug 13 17:51:11 polisnode thunderhub[22654]:        message: 'ExpectedAuthenticatedLndGrpcForGetInfoRequest',
Aug 13 17:51:11 polisnode thunderhub[22654]:        locations: [],
Aug 13 17:51:11 polisnode thunderhub[22654]:        path: [Array] },
Aug 13 17:51:11 polisnode thunderhub[22654]:      { Error: ExpectedAuthenticatedLndToRetrieveChainBalance
Aug 13 17:51:11 polisnode thunderhub[22654]:          at promise.then.catch.err (/usr/lib/thunderhub/.next/server/pages/api/v1.js:995:11)
Aug 13 17:51:11 polisnode thunderhub[22654]:          at process._tickCallback (internal/process/next_tick.js:68:7)
Aug 13 17:51:11 polisnode thunderhub[22654]:        message: 'ExpectedAuthenticatedLndToRetrieveChainBalance',
Aug 13 17:51:11 polisnode thunderhub[22654]:        locations: [],
Aug 13 17:51:11 polisnode thunderhub[22654]:        path: [Array] },
Aug 13 17:51:11 polisnode thunderhub[22654]:      { Error: ExpectedLndForPendingChainBalance
Aug 13 17:51:11 polisnode thunderhub[22654]:          at promise.then.catch.err (/usr/lib/thunderhub/.next/server/pages/api/v1.js:995:11)
Aug 13 17:51:11 polisnode thunderhub[22654]:          at process._tickCallback (internal/process/next_tick.js:68:7)
Aug 13 17:51:11 polisnode thunderhub[22654]:        message: 'ExpectedLndForPendingChainBalance',
Aug 13 17:51:11 polisnode thunderhub[22654]:        locations: [],
Aug 13 17:51:11 polisnode thunderhub[22654]:        path: [Array] },
Aug 13 17:51:11 polisnode thunderhub[22654]:      { Error: ExpectedLndGrpcApiForChannelBalanceQuery
Aug 13 17:51:11 polisnode thunderhub[22654]:          at promise.then.catch.err (/usr/lib/thunderhub/.next/server/pages/api/v1.js:995:11)
Aug 13 17:51:11 polisnode thunderhub[22654]:          at process._tickCallback (internal/process/next_tick.js:68:7)
Aug 13 17:51:11 polisnode thunderhub[22654]:        message: 'ExpectedLndGrpcApiForChannelBalanceQuery',
Aug 13 17:51:11 polisnode thunderhub[22654]:        locations: [],
Aug 13 17:51:11 polisnode thunderhub[22654]:        path: [Array] } ],
Aug 13 17:51:11 polisnode thunderhub[22654]:   networkError: null,
Aug 13 17:51:11 polisnode thunderhub[22654]:   message:
Aug 13 17:51:11 polisnode thunderhub[22654]:    'ExpectedAuthenticatedLndGrpcForGetInfoRequest\nExpectedAuthenticatedLndToRetrieveChainBalance\nExpectedLndForPendingChainBalance\nExpectedLndGrpcApiForChannelBalanceQuery',
Aug 13 17:51:11 polisnode thunderhub[22654]:   extraInfo: undefined }
Aug 13 17:51:25 polisnode thunderhub[22654]: 2020-08-13 17:51:25 warn [THUB]: No SSO account available
Aug 13 17:51:32 polisnode thunderhub[22654]: 2020-08-13 17:51:32 warn [THUB]: No SSO account available
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400,
Aug 13 17:51:39 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndGrpcForGetInfoRequest',
Aug 13 17:51:39 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400, 'ExpectedLndForInvoiceListing', [length]: 2 ]
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400, 'ExpectedLndToGetChannels', [length]: 2 ]
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400, 'ExpectedLndToGetForwardingHistory', [length]: 2 ]
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400, 'ExpectedLndToGetForwardingHistory', [length]: 2 ]
Aug 13 17:51:39 polisnode thunderhub[22654]: 2020-08-13 17:51:39 error [THUB]: [ 400, 'ExpectedLndForNetworkInfoRequest', [length]: 2 ]
Aug 13 17:54:08 polisnode thunderhub[22654]: 2020-08-13 17:54:08 error [THUB]: [ 400,
Aug 13 17:54:08 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndGrpcForGetInfoRequest',
Aug 13 17:54:08 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:54:08 polisnode thunderhub[22654]: 2020-08-13 17:54:08 error [THUB]: [ 400,
Aug 13 17:54:08 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndToRetrieveChainBalance',
Aug 13 17:54:08 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:54:08 polisnode thunderhub[22654]: 2020-08-13 17:54:08 error [THUB]: [ 400, 'ExpectedLndForPendingChainBalance', [length]: 2 ]
Aug 13 17:54:08 polisnode thunderhub[22654]: 2020-08-13 17:54:08 error [THUB]: [ 400, 'ExpectedLndGrpcApiForChannelBalanceQuery', [length]: 2 ]
Aug 13 17:54:12 polisnode thunderhub[22654]: 2020-08-13 17:54:12 error [THUB]: [ 400,
Aug 13 17:54:12 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndGrpcForGetInfoRequest',
Aug 13 17:54:12 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:54:12 polisnode thunderhub[22654]: 2020-08-13 17:54:12 error [THUB]: [ 400,
Aug 13 17:54:12 polisnode thunderhub[22654]:   'ExpectedAuthenticatedLndToRetrieveChainBalance',
Aug 13 17:54:12 polisnode thunderhub[22654]:   [length]: 2 ]
Aug 13 17:54:12 polisnode thunderhub[22654]: 2020-08-13 17:54:12 error [THUB]: [ 400, 'ExpectedLndForPendingChainBalance', [length]: 2 ]
Aug 13 17:54:12 polisnode thunderhub[22654]: 2020-08-13 17:54:12 error [THUB]: [ 400, 'ExpectedLndGrpcApiForChannelBalanceQuery', [length]: 2 ]
Aug 13 17:54:12 polisnode thunderhub[22654]: 2020-08-13 17:54:12 error [THUB]: [ 400, 'ExpectedLndForInvoiceListing', [length]: 2 ]
```

</details>

**Your environment**
* Version of ThunderHub: 0.9.3 (upgraded from 0.8.13)
* Deployment method: Debian package (upgrade)
* Other relevant environment details: LND was not restarted ans was unlocked the whole time, so it shouldn't be related to unlocking. Mainnet.

**To Reproduce**
1. Install version 0.8.13 (e.g. using Debian repository)
2. Upgrade to 0.9.3
3. See glitches
4. Look into log
5. See exception

**Expected behavior**
No exceptions, no glitches.

**Actual behavior**
Exceptions in log and glitches in web browser.

**Screenshots/Links**
Unfortunately it didn't occur to me to make screenshots and now it's working. I may try to reproduce it if the logs aren't sufficient to understand the problem.

**Comment by apotdevin** [comment]
There's some misconfiguration in the server side rendering where it seems to pass 2 times through the server and on one of the calls the lnd info isn't passed. Still haven't found the reason for this unfortunately

This still shouldn't affect the rendering of the pages (apart from the annoying logs)

**Comment by apotdevin** [comment]
Fixed on version 0.9.7 (adc935269ccad538957dba30d4cca4f26d73dff0)
